values.yaml

```diff
@@ -218,6 +218,10 @@ compilers:
     platform: ps2
     template: common/default
     file: https://cdn.discordapp.com/attachments/1067192766918037536/1120445479797395506/ee-gcc2.9-991111a.tar.xz
+  - id: ee-gcc2.9-991111b-r4
+    platform: ps2
+    template: common/default
+    file: https://github.com/TheOnlyZac/compilers/releases/download/ee-gcc2.9-991111b%2Fr4/ee-gcc2.9-991111.tar.gz
```

Comment:
I've added this file to *our* compilers release. please update the link to:
```
https://github.com/decompme/compilers/releases/download/compilers/ee-gcc2.9-991111.tar.gz
```

Reply:
Done